### PR TITLE
Include NuGet.Tests.Apex.dll in NuGet.Tests.Apex.Daily.runsettings

### PR DESCRIPTION
**Pull request by SueSu01**
## Bug

Fixes: https://github.com/NuGet/Client.Engineering/issues/2778

Regression? Last working version:

## Description

Add all other existing Apex test cases to the new daily pipeline and in order to avoid the risk of duplication so include the NuGet.Tests.Apex.dll in NuGet.Tests.Apex.Daily.runsettings.

## PR Checklist

- [x] PR has a meaningful title
- [x] PR has a linked issue.
- [x] Described changes

- **Tests**
  - [ ] Automated tests added
  - **OR**
  
  - [ ] Test exception
  - **OR**
  - [x] N/A 

- **Documentation**
  
  - [ ] Documentation PR or issue filled
  - **OR**
  - [x] N/A

### CODE REVIEW

File: build/template.runsettingsproj [hunk 1]
```diff
@@ -49,6 +49,7 @@
 
   <ItemGroup Label="Test Containers">
     <TestContainer Include="$(FileName)" />
+    <TestContainer Include="NuGet.Tests.Apex.dll" Condition=" '$(RunName)' == 'NuGet.Tests.Apex.Daily' " />
```
Comment (nkolev92) [inline build/template.runsettingsproj]: Can this be passed from runsettings.proj instead?

FileName is our own property, so we can make different, like for example, we can split by `,` or something and include all of that as a test container.

Comment (SueSu01) [reply build/template.runsettingsproj]: > Can this be passed from runsettings.proj instead?
> 
> FileName is our own property, so we can make different, like for example, we can split by `,` or something and include all of that as a test container.

@nkolev92 I tried split the multiple .dlls with `&` and ` ,` respectively and assign it to FileName, the run failed at `Deploy Tests Install-TestContainers` . 
image
I'm not sure if a test container can include mutiple dlls. Do you have any suggesstion or idea?

Comment (zivkan) [reply build/template.runsettingsproj]: MSBuild syntax is to use a semicolon: https://learn.microsoft.com/en-us/visualstudio/msbuild/how-to-select-the-files-to-build?view=vs-2022#to-declare-multiple-items

Comment (SueSu01) [reply build/template.runsettingsproj]: @zivkan @nkolev92 Thanks for your guidance. Updated the code with your suggestion.

**Review by nkolev92 — APPROVED** [review]

File: build/template.runsettingsproj (hunk 1 above)
Comment (zivkan) [reply build/template.runsettingsproj]: Looking at the daily test pipeline, I don't see a manual run of this branch to validate that this PR works as intended.  Have you tested this change somewhere else?

Comment (SueSu01) [reply build/template.runsettingsproj]: > Looking at the daily test pipeline, I don't see a manual run of this branch to validate that this PR works as intended. Have you tested this change somewhere else?

I verified that this PR works on this [daily run](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9445583&view=results) triggered by [private build](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9445925&view=results) which include my changes.

Comment (zivkan) [reply build/template.runsettingsproj]: @SueSu01 that CI run looks like it only ran the apex tests, but not the daily tests assembly. So, it only ran one of the two test assemblies.

Comment (SueSu01) [reply build/template.runsettingsproj]: > @SueSu01 that CI run looks like it only ran the apex tests, but not the daily tests assembly. So, it only ran one of the two test assemblies.

Currently daily tests include 42 P0 and P1 apex cases. The NuGet.Tests.Apex.dll assembly has 101 apex tests and I checked that CI run has 143 tests totally, so it ran two test assemblies.

Comment (nkolev92) [reply build/template.runsettingsproj]: image

This shows the multiple assemblies were run too. 

Conveniently there was a failure in each :D

Comment (v-luzh) [reply build/template.runsettingsproj]: Yes, we will try to investigate the failures in the recent runs during the following week.

**Pull request merged**